test/interfaces/test_midi.py

```diff
@@ -18,6 +18,7 @@
     mido_backend_error = str(e)
 
 
+@unittest.skipUnless(mido_backend_available, "mido MIDI backend is not available: {}".format(mido_backend_error))
```

Comment:
This line is new to skip this test as well.  Other `@unittest.skipUnless` changes is a typo fix only ("available").